contrib/core/actions/uuid.yaml

```diff
@@ -0,0 +1,6 @@
+---
+name: uuid
```

Comment:
Just thinking out loud - would we ever want to add support for other types of UUID? If so it, might make sense to call this action `uuid4`, but in any case, not a big deal - we can always change it in the future in a backward compatible manner if necessary.